engine/src/test/java/com/datatorrent/stram/engine/AutoMetricTest.java

```diff
@@ -361,6 +394,31 @@ public int getMyMetric()
     }
   }
 
+  public static class OperatorAndAggregator extends OperatorWithMetrics implements AutoMetric.Aggregator, Serializable
```

Comment:
This is because OperatorMeta needs to be Serializable. OperatorMeta contains MetricsAggregatorMeta which comprises of AutoMetric.Aggregator and AutoMetric.DimensionScheme.